public/app/views/shared/_result.html.erb

```diff
@@ -63,10 +63,24 @@
       <div class="result_context">
         <strong><%= t('context') %>: </strong>
         <span  class="repo_name">
+          <%= badge_for_type('repository') %>
           <%= link_to result.resolved_repository.fetch('name'),
                      app_prefix(repository_base_url(
                       "uri" => result.resolved_repository.fetch('uri'),
                       "slug" => result.resolved_repository.fetch('slug') {|s| nil })) %>
+          <% if result.is_a?(DigitalObject) %>
+            <% resolved_instances_record = 
+                 ArchivesSpaceClient.instance.get_record(result.uri, {'resolve[]' => ['linked_instance_uris:id']}) %>
```

Comment:
Do we want to make an http request to fetch template data from the view layer, or send the data to the template from a controller?